app/src/index.tsx

```diff
@@ -2,11 +2,5 @@ import React from 'react'
 import ReactDOM from 'react-dom'
 
 import App from './App'
-import * as serviceWorker from './serviceWorker'
```

Comment:
@hexyls 
Why did you remove serviceWorker?

Reply:
haha oh yeah I noticed we aren't using it (we unregister the service worker immediately [here](https://github.com/protofire/omen-exchange/pull/1215/files/0f51a2513eb17744690a0ee08924687306a189fc#diff-f2e29ede120f9b093448582666e7c0eab8fd03e8e69f7522e5011901c54819ddL12)) so we are loading that extra code for no reason.

Reply:
@hexyls , don't remove serviceWorker. It can be necessary later.
And this issue(PR) is not related to serviceWorker.